lib/irc/ClientPool.js

```diff
@@ -280,14 +280,18 @@ ClientPool.prototype._onClientDisconnected = function(bridgedClient) {
         cliConfig, bridgedClient.matrixUser, bridgedClient.isBot
     );
     var chanList = bridgedClient.chanList;
-    var self = this;
     // remove ref to the disconnected client so it can be GC'd. If we don't do this,
     // the timeout below holds it in a closure, preventing it from being GC'd.
     bridgedClient = undefined;
-    this._reconnectQueue.enqueue(cli._id, {cli, chanList});
+    this._reconnectQueue.enqueue(cli._id, {
+        cli: cli,
+        chanList: chanList,
+    });
 };
 
-ClientPool.prototype._reconnectClient = function({cli, chanList}) {
+ClientPool.prototype._reconnectClient = function(cliChan) {
+    const cli = cliChan.cli;
+    const chanList = cliChan.chanList;
```

Comment:
What's the point of this?

Reply:
eslint 1 doesn't understand the syntax and I don't want to bump it and break more things in *this* PR.

Reply:
okie dokie

Reply:
ouch.. eslint 1??